cmd/mattermost/commands/config.go

```diff
@@ -102,3 +116,143 @@ func configSubpathCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func configGetCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// return error if number of arguments is zero or more than one
+	if len(args) == 0 {
+		return fmt.Errorf("failed to enter the name of the config setting")
+	} else if len(args) > 1 {
+		return fmt.Errorf("only one config setting can be given to the get command")
+	}
+
+	// create the model for config
+	// Note: app.Config() returns a pointer, make appropriate changes
+	config := app.Config()
+
+	// find the path of the config file
+	filePath := utils.FindConfigFile("config.json")
+
+	// open the file
+	file, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+
+	// create a decoder on the file
+	decoder := json.NewDecoder(file)
+
+	// decode the file into our model
+	err = decoder.Decode(config)
+	if err != nil {
+		return err
+	}
+
+	// print out the config setting and return error
+	return printConfigValues(configToMap(*config), strings.Split(args[0], "."), args[0])
+}
+
+// printConfigValues function prints out the value of the configSettings working recursively or
+// gives an error if config setting is not in the file.
+func printConfigValues(configMap map[string]interface{}, configSetting []string, name string) error {
```

Comment:
I would expect same approach here, return (string, error), and only print directly from the command function.